docs/source/Developer/Coding_standards.rst

```diff
@@ -0,0 +1,125 @@
+.. _coding_standards:
+
+============
+Contributing
```

Comment:
can you rename the folders/files in small letters, please:
e.g. Developer --> developer
Coding_standards.rst --> coding_standards.rst
(since we are already enforcing a rule on file names)